packages/trpc/server/routers/loggedInViewer/outOfOffice.handler.ts

```diff
@@ -50,6 +50,14 @@ export const outOfOfficeCreate = async ({ ctx, input }: TBookingRedirect) => {
     const user = await prisma.user.findUnique({
       where: {
         id: input.toTeamUserId,
+        /** You can only create OOO for members of teams you're either ADMIN or OWNER */
+        teams: {
```

Comment:
shouldn't we rather check here if `ctx.user.id` and `toTeamUserId` are in the same teams
?

Reply:
RN we're checking if `toTeamUserId` belongs to a team where the `ctx.user.id` is either an ADMIN or OWNER. Isn't that what we want?

Reply:
I see. Then we can either add MEMBER to the role check or skip the role check altogether.

Reply:
I see. Then we can either add MEMBER to the role check or skip the role check altogether.

Reply:
Addressed here [b9ac480](https://github.com/calcom/cal.com/pull/14736/commits/b9ac480c42c0664a6f6388621ace4a0f01fe171f)